website/src/components/EventSlider/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import Slider from "react-slick";
+import { useTranslation } from "react-i18next";
+import { Typography, Link, Box } from "@material-ui/core";
+import useStyles from "./style";
+import events from '../../resources/events.json';
+
+const EventSlider: React.FC = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  const NextArrow = (props: any) => {
+    const { className, onClick } = props;
+    return (
+      <div
+        className={`${className} ${classes.slickButtons}`}
+        onClick={onClick}
+      >
+        <img
+          src="../Images/svg/right_arrow.svg"
+          alt={t("home.adaptorsTestimonials.nextArrowAlt")}
```

Comment:
This alt text says `Next testimonial`. Can we add a new translation key for events and use it here?